### DISCUSSION TITLE
The user I have granted access to my personal project is not notified and cannot see it without an explicit link

### DISCUSSION THREAD

**Discussion by gwesterman** [community/community#45856] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

I have a personal repository with a personal project linked to it. I have added another person as a collaborator with write permission to this project by searching for their username and pressing invite. It shows me that they have been added with the specified role. On my end everything looks fine.

But they never receive any kind of notification. Neither an email, or a a notification on GitHub itself. The project does not appear in their project view. I had to send them the link to the project in order for them to access it.

Is this how personal projects should work and would it be different if the project belonged to an organization that the user belonged to? Does this in turn mean I need to transfer my personal repository over to an organization?

**Comment by cezarBdz** [comment]
I have the same issue, any updates on that case?

**Comment by AdityaRanjanSingh** [comment]
I am facing a similar issue, Cannot invite collaborator to project

**Comment by jearbecerro** [comment]
You can check you email, its there.
